### ISSUE TITLE
Expand pp checking  

### ISSUE THREAD

**Issue by jgabry** [stan-dev/shinystan#67]
First version of pp checking will be included in v1.1.0 to be released soon. 

Opening this issue to keep improving and expanding pp checking capabilities on the to-do list.

**Comment by bgoodri** [comment]
Also, for a `stanreg` object, shinystan should just call `posterior_predict` rather than asking the user to select the name of the posterior predictions.

**Comment by jgabry** [comment]
Yes, good call. I will put this in the next release. Thoughts on particular
pp checking plots for rstanarm models?

If I really wanted to put some time into it I could have
shinystan automatically adjust various settings and plots for the different
rstanarm models using the info in the stanreg object to classify the model.

**Comment by bgoodri** [comment]
I don't really know; start with just making it possible to do something
with a `stanreg` object because it is not going to have a drop-down entry
for y_rep. Rather than investing a lot of time customizing for `stanreg`
objects, it seems that you could do a fair amount of reasonable
customization just based on the class of the dependent variable.

**Comment by jgabry** [comment]
Yeah, definitely. Recognizing the dependent variable type and selecting the
content accordingly is one of the things I've been wanting to do.

**Comment by bgoodri** [comment]
At a minimum, a `stanreg` object can store its y in a canonical form for
each type of model so it is easy to use if shinystan knows those forms.
